OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -82,6 +82,9 @@ UCM_CLIENT_SECURITY_POLICY=<required if jaxws>
 # Elasticsearch Server Configuration details
 SEARCH_APP_USERNAME=<Search User Name>
 SEARCH_APP_USER_PASSWORD=<Search User Password>
+ELASTIC_SEARCH_INSTALLER_FILE_NAME=<Elasticsearch Installer File Name>
+ELASTIC_SEARCH_VERSION=<Elasticsearch Version>
```

Comment:
```suggestion
ELASTICSEARCH_INSTALLER_FILE_NAME=<Elasticsearch Installer File Name>
ELASTICSEARCH_VERSION=<Elasticsearch Version>
```

Reply:
> 

Webcenter Portal expects Elastic search input variable with underscore. Please refer 
https://docs.oracle.com/en/middleware/webcenter/portal/12.2.1.4/admin/managing-search-webcenter-portal-elasticsearch.html#GUID-15836948-5875-4216-96C1-3985ECB21C4C

From Readme I can remove the underscore but while passing input to webcenter we need to pass the input variable as mentioned in above link